diff --git a/src/common/read_config.c b/src/common/read_config.c
index 492109f..8446488 100644
--- a/src/common/read_config.c
+++ b/src/common/read_config.c
@@ -3069,8 +3069,15 @@ static int _init_slurm_conf(const char *file_name)
 
 	if (name == NULL) {
 		name = getenv("SLURM_CONF");
-		if (name == NULL)
+		if (name == NULL) {
+			struct stat stat_buf;
 			name = default_slurm_config_file;
+			/* default_slurm_config_file /etc/slurm/slurm.conf may not exist.
+			 * If it does, use it, if not, use the "stateless" file */
+			if (stat(name, &stat_buf) == -1) {
+				name = "/usr/share/defaults/slurm/slurm.conf";
+			}
+		}
 	}
 	if (conf_initialized)
 		error("the conf_hashtbl is already inited");
@@ -5825,8 +5832,15 @@ extern char *get_extra_conf_path(char *conf_name)
 	char *val = getenv("SLURM_CONF");
 	char *rc = NULL, *slash;
 
-	if (!val)
+	if (!val) {
+		struct stat stat_buf;
 		val = default_slurm_config_file;
+		/* default_slurm_config_file /etc/slurm/slurm.conf may not exist.
+		 * If it does, use it, if not, use the "stateless" file */
+		if (stat(val, &stat_buf) == -1) {
+			val = "/usr/share/defaults/slurm/slurm.conf";
+		}
+	}
 
 	/*
 	 * Both plugstack.conf and topology.conf need special handling in
